server/ec2spotmanager/urls.py

```diff
@@ -18,12 +18,12 @@
     url(r'^pools/(?P<poolid>\d+)/disable/$', views.disablePool, name='pooldisable'),
     url(r'^pools/(?P<poolid>\d+)/cycle/$', views.forceCyclePool, name='poolcycle'),
     url(r'^pools/messages/(?P<msgid>\d+)/delete/$', views.deletePoolMsg, name='poolmsgdel'),
-    url(r'^configurations/$', views.viewConfigs, name='configs'),
-    url(r'^configurations/create/$', views.createConfig, name='configcreate'),
-    url(r'^configurations/(?P<configid>\d+)/$', views.viewConfig, name='configview'),
-    url(r'^configurations/(?P<configid>\d+)/edit/$', views.editConfig, name='configedit'),
-    url(r'^configurations/(?P<configid>\d+)/delete/$', views.deleteConfig, name='configdel'),
-    url(r'^configurations/(?P<configid>\d+)/cycle/$', views.forceCyclePoolsByConfig, name='configcycle'),
+    url(r'^Ec2configurations/$', views.viewEc2Configs, name='configs'),
+    url(r'^Ec2configurations/create/$', views.createEc2Config, name='configcreate'),
+    url(r'^Ec2configurations/(?P<configid>\d+)/$', views.viewEc2Config, name='configview'),
+    url(r'^Ec2configurations/(?P<configid>\d+)/edit/$', views.editEc2Config, name='configedit'),
+    url(r'^Ec2configurations/(?P<configid>\d+)/delete/$', views.deleteEc2Config, name='configdel'),
+    url(r'^Ec2configurations/(?P<configid>\d+)/cycle/$', views.forceCyclePoolsByConfig, name='configcycle'),
```

Comment:
The URLs should not be changing wholesale. As much as possible, we should not duplicate everything for each provider.